### ISSUE TITLE
wanted cross-context recognition

### ISSUE THREAD

**Issue by jbradleychen** [w3ctag/privacy-principles#109]
https://github.com/w3ctag/privacy-principles/blob/0757d6fbeb31b17f5d12c55c80d7140ecc9a610b/index.html#L707

What allowances should be made for wanted/desirable cross-context recognition? Regrettably, as siloed defense of large platforms has improved, cross-platform threat models have become common. Specific situations where cross-context recognition is relevant for safety:
- cross-platform threat models. 
-- Example: an adult impersonating a minor solicits a minor on a large public platform, then vectors the discussion to an E2EE platform where criminal activity can occur in secret.
-- Example: analogous scenarios for fraud, malware.
-- Example: bypassing content moderation on platform A by exploiting inbound traffic from platform B.
- legitimate law enforcement referrals.
- intra-platform defenses stymied by the annihilation of signals potentially useful for cross-platform correlation.

When do nuisance advertising and the other harms of imperfect privacy justify disrupting the ability of safety efforts to deal with fraud, disinformation, malware, exploitation? How do we comprehend balance?

A basic problem with the line of thinking in this section is that it fails to differentiate the growth and revenue generating parts of online services from legitimate public safety efforts. I think we may need to develop a strategy that properly limits the data available for growth, while giving public safety work access to a larger set of data. Consider this <a href="https://datatracker.ietf.org/doc/draft-ip-address-privacy-considerations/">IETF PEARG draft</a> and the notion of replacement signals for counterabuse.

**Comment by dmarti** [comment]
I agree that it is important to leave out any mentions of nuisance advertising or similar annoyances. Annoying UX is not necessarily a privacy issue (although it can be, when dealing with it requires a person to do additional privacy labour.)

Legitimate public safety efforts are important, and in many cases those efforts will require improved privacy for service members and other key people. See [Microtargeting as Information Warfare](https://cyberdefensereview.army.mil/Portals/6/Documents/2021_winter_cdr/04_CDR_V6N1_Dawson.pdf?ver=_cSYLxO5k8EHPSmFziZU3g%3d%3d) by Dr. Jessica Dawson.
